### PR TITLE
draft for mirdata rtd. including example, faq(placeholder)

### PR DESCRIPTION
**Pull request by keunwoochoi**

### CODE REVIEW

File: docs/source/example.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+The ID's and annotation data can be loaded as below.
+
+.. code-block:: python
+    :linenos:
+
+    # Load the dataset
+    orchset_data = mirdata.orchset.load()
+
+    orchset_ids = mirdata.orchset.track_ids()
+    orchset_data = mirdata.orchset.load()
```
Comment (rabitt) [inline docs/source/example.rst]: duplicated from line 46

File: docs/source/example.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    # todo: add __str__() method and print(orchset_data)
+
+
+A more end-to-end example would be like this.
```
Comment (rabitt) [inline docs/source/example.rst]: "If we wanted to use Orchset to evaluate the performance of a melody extraction algorithm (in our case, `very_bad_melody_extractor`), and then split the scores based on the metadata, we could do the following:"

File: docs/source/example.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                   ('Raw Chroma Accuracy', 0.07377049180327869),
+                   ('Overall Accuracy', 0.008754863813229572)]),
+
+    ...
+
+      'Wagner-Tannhauser-Act2-ex2': OrderedDict([('Voicing Recall', 1.0),
+               ('Voicing False Alarm', 1.0),
+               ('Raw Pitch Accuracy', 0.03685636856368564),
+               ('Raw Chroma Accuracy', 0.08997289972899729),
+               ('Overall Accuracy', 0.036657681940700806)])}}
```
Comment (rabitt) [inline docs/source/example.rst]: `very_bad_melody_extractor` performs very badly!

**Comment by keunwoochoi** [comment]
Hm, I'm not sure why the above two take like forever?

**Comment by rabitt** [comment]
Merging!

**Pull request merged**